src/server/raft_client.rs

```diff
@@ -170,9 +172,10 @@ impl RaftClient {
             }
 
             counter += 1;
-            let mut msgs = conn.buffer.take().unwrap();
-            msgs.last_mut().unwrap().1 = WriteFlags::default();
-            if let Err(e) = conn.stream.unbounded_send(msgs) {
+
+            let mut batch_msgs = BatchRaftMessage::new();
```

Comment:
do we need to control the total batch size, not too large?

Reply:
Here we won't wait any coming messages, so it's not needed?